cmd/snap-update-ns/main.go

```diff
@@ -42,13 +44,19 @@ func main() {
 func parseArgs(args []string) error {
 	parser := flags.NewParser(&opts, flags.HelpFlag|flags.PassDoubleDash|flags.PassAfterNonOption)
 	_, err := parser.ParseArgs(args)
-	return err
+	if err != nil {
+		return err
+	}
+	return snap.ValidateName(opts.Positionals.SnapName)
 }
 
 func run() error {
 	if err := parseArgs(os.Args[1:]); err != nil {
 		return err
 	}
-
+	if err := bootstrapError(); err != nil {
```

Comment:
Maybe a small comment here that that the bootstrap C code is automatically called and that we need to check errors for that here?